src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  const basicDiscount = 20;
+  const fullDiscount = 50;
+  let total = 0;
```

Comment:
```suggestion
  const basicPrice = rent * days;
```
It would be great to create a variable for `rent * days` in order not to duplicate yourself.